web_external/js/views/body/StudyResultsView.js

```diff
@@ -0,0 +1,483 @@
+            annotation: this.annotation,
+            featureset: this.featureset,
+            parentView: this
+        });
+
+        this.localFeaturesView = new isic.views.StudyResultsLocalFeaturesView({
+            annotation: this.annotation,
+            featureset: this.featureset,
+            parentView: this
+        });
+
+        this.studies.fetch();
+
+        this.listenTo(this.selectStudyView, 'changed', this.studyChanged);
+        this.listenTo(this.selectImageView, 'changed', this.imageChanged);
+        this.listenTo(this.selectUserView, 'changed', this.userChanged);
+
+        this.render();
+    },
+
+    studyChanged: function (studyId) {
+        this.images.reset();
+        this.users.reset();
+
+        delete this.imageId;
+        delete this.userId;
+
+        this.selectImageView.setEnabled(false);
+        this.selectUserView.setEnabled(false);
+
+        this.annotation.clear();
+        this.featureset.clear();
+
+        // Fetch selected study
+        this.study = new isic.models.StudyModel({
+            _id: studyId
+        }).once('g:fetched', function () {
+            // Populate images collection
+            var imageModels = _.map(this.study.get('images'), function (image) {
+                return new isic.models.ImageModel(image);
+            });
+            this.images.reset(imageModels);
+
+            // Populate users collection
+            var userModels = _.map(this.study.get('users'), function (user) {
+                return new girder.models.UserModel(user);
+            });
+            this.users.reset(userModels);
+
+            // Fetch featureset
+            var featureset = new isic.models.FeaturesetModel(this.study.get('featureset'));
+            featureset.once('g:fetched', function () {
+                this.featureset.set(featureset.attributes);
+            }, this).fetch();
+        }, this).fetch();
+    },
+
+    imageChanged: function (imageId) {
+        this.imageId = imageId;
+        this.annotation.clear();
+        this.fetchAnnotation();
+    },
+
+    userChanged: function (userId) {
+        this.userId = userId;
+        this.annotation.clear();
+        this.fetchAnnotation();
+    },
+
+    fetchAnnotation: function () {
+        if (!_.has(this, 'imageId') ||
+            !_.has(this, 'userId')) {
+            return;
+        }
+
+        var annotations = new isic.collections.AnnotationCollection();
+        annotations.once('g:changed', function () {
+            if (!annotations.isEmpty()) {
+                // Fetch annotation detail
+                this.annotation.set(annotations.first().attributes).fetch();
+            }
+        }, this).fetch({
+            studyId: this.study.id,
+            userId: this.userId,
+            imageId: this.imageId
+        });
+    },
+
+    render: function () {
+        this.$el.html(isic.templates.studyResultsPage({
+            title: 'Study Results'
```

Comment:
I changed this page to always pass titles to templates, similar to the other pages (e.g. Featuresets).